src/contacts/view/ContactViewer.ts

```diff
@@ -67,6 +70,38 @@ export class ContactViewer implements ClassComponent<ContactViewerAttrs> {
 						),
 						this.hasBirthday(contact) ? m("", this.formattedBirthday(contact)) : null,
 					]),
+					contact && attrs.editAction && attrs.deleteAction
+						? m(
+								".flex-end",
+								m(
+									IconButton,
+									attachDropdown({
+										mainButtonAttrs: {
+											title: "more_label",
+											icon: Icons.More,
+										},
+										childAttrs: () => {
+											return [
+												{
+													label: "edit_action",
+													icon: Icons.Edit,
+													click: () => {
+														assertNotNull(attrs.editAction)(contact)
```

Comment:
adding a static message to the assertNotNull call makes it even easier to find if something goes wrong